acmwebsite/templates/master.xhtml

```diff
@@ -12,7 +12,7 @@
   </head>
 
   <body>
-    <!-- Navbar -->
+    <!--! Navbar -->
```

Comment:
These comments were here to let you easily navigate the HTML output of the page (were not necessarily just comments for the XML markup)... please revert.